meetings/2022-10-26.md

```diff
@@ -72,6 +98,23 @@ URL: https://github.com/solid/specification/issues/462
 
 * SC: Would like to get a sense. Where to draw the line (for now). High-level view.
 * SC: Certainly technically possible. This particular discussion is not about how.
+* HS: I've implemented parts of Memento in [Reactive Solid](https://github.com/co-operating-systems/Reactive-SoLiD). There are it seems to me two parts to this problem: 1) the server needs to say it can support some of these features, and 2) a way for client to ask to set those features. This is similar to access control: A server can specify that it supports setting ACLs via a link header, allowing the client to set it by writing an access control rule. It seems we have the same here: we need some way for the server to say this container can do memento, and a way for the client then who wants to and is allowed to set that feature. This needs to be solved in a generic feature.
+* SC: Depends on what "metadata" we refer to. Response metadata or auxiliary resource that is referring to the primary resource (description resource metadata). Could be the metadata in response headers. LDP has interaction models, link with relation in header
+* HS: An extra example is one that came up recently: say you need a resource to have a certain time to live - the server  cannot really know when the next update should be. It is likely this is something the client writing the resource will know best. Is this a resource with a day's time to live, or a year? We have a pattern similar to access control. You always have the server be as generic as possible.  All a server can do is specify its abilities somewhere allowing a client to read the servers abilities. 
+* SC: Example: do we want to allow an client to instruct a server to set redirect rules. People change WebIDs, resource move, and that comes up... if you want to be able to move stuff. Is this within the scope of the protocol? Def in the scope of web architecture. The alternative is: we don't want clients to have that type of control, the server should have control of those resources, whether through a link in the headers or redirects. 
+* HS: Everything normal servers (e.g. Apache) can do, we should be able to do too. Every thing you can set by hand on the server should be available on LDP/Solid. Not every server will be able to implement all these features, so you need the server to be able to declare what a client is allowed to do. We should come up with a general solution to edit features so that once the protocol is set the rest can be done using ontologies (i.e. specific vocabularies).
+* MB: I agree with HS, it would be interesting to be able to define redirection rules, but it should be taken care in a different specification, not the Solid protocol. Maybe a new kind of auxiliary resource. 
```

Comment:
```suggestion
* MB: I agree with HS, it would be interesting to be able to define redirection rules, but it should be taken care of in a different specification, not the Solid protocol. Maybe a new kind of auxiliary resource. 
```